src/Uno.UI/UI/Xaml/Media/Imaging/Svg/ISvgProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿#nullable enable
+
+using System;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+
+namespace Uno.UI.Xaml.Media.Imaging.Svg;
+
+public interface ISvgProvider
```

Comment:
Does it need to be public ?

Reply:
@jeromelaban It needs to be public because it is API extension interface (afaik the App.xaml.cs code generation will generate the ApiExtensibility.Register calls). Theoretically could make just a "marker" interface with no members and make an `internal` interface with the actual members, if that would be preferrable

Reply:
Got it! No that's fine to keep it that way, though documenting it is required in this case.